magento/magento2
# Add translatable text in address renderers

### Description (*)
Added translate tags around text that was not able to translate yet for address renderers


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#36942

### Manual testing scenarios (*)

1. Add translations for "VAT" other than English and switch store to that language.
2. Add product to cart
3. Go to the checkout fill in your address including VAT.
4. In the payment/billing step you will see that VAT has the translation that you have set

### Questions or comments


### Contribution checklist (*)
